src/docs/release/breaking-changes/image-cache-and-provider.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Changes to ImageCache and ImageProvider
+description: ImageCache requires implementers to override containsKey, and
+ImageProvider has marked resolve as @nonVirtual
+---
+
+
+## Summary
+
+`ImageCache` now has a method called `obaintsKey`. `ImageProvider` subclasses
```

Comment:
obaintsKey => containsKey. ??

Reply:
Whups.  Yes.  There is also a method calld `obtainKey`, must've had a brain fart in there haha